### ISSUE TITLE
OpenCV can't install needs some extra $(inherited)

### ISSUE THREAD

**Issue by nissaba** [twelve17/openalpr-ios#6]
Tried to get this to work, but I keep getting this error: 

[!] The `openalpr-xcode [Debug]` target overrides the `HEADER_SEARCH_PATHS` build setting defined in `Pods/Target Support Files/Pods/Pods.debug.xcconfig'. This can lead to problems with the CocoaPods installation
    - Use the `$(inherited)` flag, or
    - Remove the build settings from the target.

[!] The `openalpr-xcode [Release]` target overrides the `HEADER_SEARCH_PATHS` build setting defined in `Pods/Target Support Files/Pods/Pods.release.xcconfig'. This can lead to problems with the CocoaPods installation
    - Use the `$(inherited)` flag, or
    - Remove the build settings from the target.

I added the $(inherited) to the "Other linker flags" as suggested by some people but still no hope. the project does have the "$(inherited)" set in the header search paths..

**Comment by nissaba** [comment]
post adding the "$(inherited)" I am getting some pod install trouble  
ended up using the following pod file 

--------

platform :ios, '7.0'

source 'https://github.com/CocoaPods/Specs.git'

target "openalpr-xcode" do
    pod 'OpenCV'
end

----------

**Comment by twelve17** [comment]
Hi @nissaba -- thanks for your feedback, and sorry for the delay.

For v2 of this project, I have replaced the old instructions and bash scripts with a Ruby script to build the OpenALPR and dependency Frameworks, as well as the OpenALPR Xcode project.  A sample app is included.  These changes have been pushed to the master branch, and the old version has been moved to branch `1_0`.  Feel free to check out this new script and see if it works for you.  I will no longer be working on or supporting the `1_0` codebase.
